public/components/custom_panels/home.tsx

```diff
@@ -23,11 +25,37 @@ import {
   OBSERVABILITY_BASE,
   SAVED_OBJECTS,
 } from '../../../common/constants/shared';
-import { CustomPanelListType } from '../../../common/types/custom_panels';
+import { CustomPanelListType, VisualizationType } from '../../../common/types/custom_panels';
 import { ObservabilitySideBar } from '../common/side_nav';
 import { CustomPanelTable } from './custom_panel_table';
 import { CustomPanelView } from './custom_panel_view';
 import { isNameValid } from './helpers/utils';
+import { SavedObject } from '../../../../../src/core/types';
+import { CustomPanelViewSO } from './custom_panel_view_so';
+// import { ObjectFetcher } from '../common/objectFetcher';
+
+export interface ObservabilitySavedObjectAttrs<T> {
+  id: string;
+  name: string;
+  attributes: T;
+}
+
+export interface ObservabilityPanelAttrs {
+  title: string;
+  description: string;
+  dateCreated: number;
+  dateModified: number;
+  timeRange: {
+    to: string;
+    from: string;
+  };
+  queryFilter: {
+    query: string;
+    language: string;
+  };
+  visualizations: VisualizationType[];
+  applicationId: string;
+}
```

Comment:
Should this be moved to common/types?

Reply:
Maybe.  So far they're only used on Home.tsx